geometric_kernels/kernels/geometric_kernels.py

```diff
@@ -79,13 +79,13 @@ def _spectrum(
         Depends on the `lengthscale` parameters.
         """
         if nu == np.inf:
-            return B.exp(-(lengthscale**2) / 2.0 * from_numpy(lengthscale, s**2))
+            return B.exp(-(lengthscale ** 2) / 2.0 * from_numpy(lengthscale, s ** 2))
         elif nu > 0:
             power = -nu - self.space.dimension / 2.0
-            base = 2.0 * nu / lengthscale**2 + B.cast(
-                B.dtype(nu), from_numpy(nu, s**2)
+            base = 2.0 * nu / lengthscale ** 2 + B.cast(
```

Comment:
Here and so on, you sure you've got the most up-to-date linter? Because in the main branch, it is always `a**b`, never `a ** b` and the linter complains about the latter